yt/data_objects/static_output.py

```diff
@@ -330,6 +330,9 @@ def __reduce__(self):
         return (_reconstruct_ds, args)
 
     def __repr__(self):
+        return f"{self.__class__.__name__}: {self.parameter_filename}"
```

Comment:
Any reason not to do something like
```suggestion
        return f"{self.__class__.__name__}(filename={self.parameter_filename})"
```
This would allow an almost 1-to-1 mapping between the way the class was initialised and how it is `repr`esented.

Reply:
I don't think this is a bad idea on paper, your suggestions hits a problem that was raised by @munkm as https://github.com/yt-project/yt/issues/2879, so I don't think it's worth doing it this way for now.

This __repr__ should be seen as an implementation detail however, and I do think it can be further improved in the future since any downstream code relying on it should be considered broken IMO.

Reply:
Fair enough; I'm fine with it as is